spec/fpm/package/deb_spec.rb

```diff
@@ -362,13 +366,13 @@ def dpkg_field(field)
     context "when run against lintian", :if => have_lintian do
       lintian_errors_to_ignore = [
         "no-copyright-file",
-        "init.d-script-missing-lsb-section",
-        "non-standard-file-permissions-for-etc-init.d-script"
+        "script-in-etc-init.d-not-registered-via-update-rc.d"
       ]
 
       it "should return no errors" do
-        lintian_output = `lintian #{target} --suppress-tags #{lintian_errors_to_ignore.join(",")}`
+        lintian_output = `lintian #{target} --suppress-tags '#{lintian_errors_to_ignore.join(",")}'`
         expect($CHILD_STATUS).to eq(0), lintian_output
+        expect($CHILD_STDERR).to be nil
```

Comment:
Ah, maybe too restrictive, I didn't noticed the test title "should return no error". Should I revert it or it's acceptable to leave the test in this stricter version?